lib/completion.js

```diff
@@ -44,7 +44,11 @@ module.exports = function (yargs, usage, command) {
     var handlers = command.getCommandHandlers()
     for (var i = 0, ii = args.length; i < ii; ++i) {
       if (handlers[args[i]] && handlers[args[i]].builder) {
-        return handlers[args[i]].builder(yargs.reset()).argv
+        const builder = handlers[args[i]].builder
```

Comment:
this logic was creating an exception that we were quietly ignoring. It came up when the builder was an object rather than a function, or the builder returned nothing -- because this error wasn't a `YError` tests started failing.